frontend/src/pages/team/Instances.vue

```diff
@@ -77,6 +88,17 @@
                         </ff-button>
                     </template>
                 </EmptyState>
+                <EmptyState v-else>
+                    <template #img>
+                        <img src="../../images/empty-states/team-instances.png">
+                    </template>
+                    <template #header>There are currently no instances with dashboards in this team</template>
+                    <template #message>
+                        <p>
+                            Ask an Admin or Team Owner to verify the Instances you should have access to.
+                        </p>
+                    </template>
```

Comment:
I would be tempted to leave this as "There are no dashboards in this team"

Not sure the subtitle is necessary or even correct.  If there were to be a subtitle, it might be better as something like "Ask an Admin or Team Owner install the FlowFuse Dashboard on an instances in this team" - but that doesnt feel right.  I _think_ the suggested simpler title is more succinct and to the point.